web-app/src/components/Header.js

```diff
@@ -18,13 +20,38 @@ const useStyles = makeStyles((theme) => ({
 
 export default function Header() {
 	const classes = useStyles();
+	const [state, setState] = useState({
```

Comment:
The switch value should be saved in the context so that it can be used to switch between staging and production base url